lab_1_classify_by_unigrams/main.py

```diff
@@ -11,14 +11,36 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
 
+    list_of_tokens = ""
+    for token in text:
+        if token.isalpha():
+            new_token = token.lower()
+            list_of_tokens += new_token
+    tokens = list(list_of_tokens)
+
+    return tokens
 
 def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     """
     Calculates frequencies of given tokens
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
+    if not isinstance(tokens, list) or not all(isinstance(letter, str) for letter in tokens):
+        return None
+
+    list_of_tokens = ""
```

Comment:
for what?

Reply:
fixed (if i understood correctly)

Reply:
do not think so

Reply:
now should be fixed 100% (i corrected it i don't know why nothing changed at first time :c )